### DISCUSSION TITLE
How to set precision when using dump()

### DISCUSSION THREAD

**Discussion by sunshichao520** [nlohmann/json#4065] [Q&A] [answered]
Code like this:
 ```
   std::vector<double> double_data = {0.1234567234123, 0.123456231345, 0.132345346, 0.234345341123};
    nlohmann::json json;
    json["test"] = double_data;
    std::string ss = json.dump();
```

output string is:
{"test":[0.1234567234123, 0.123456231345, 0.132345346, 0.234345341123]}

I want to set the precision of a floating-point number to 3 decimal places. So I suppose the output string should be:
{"test":[0.123, 0.123, 0.132, 0.234]}

How can I do this?

`

**Comment by sunshichao520** [comment]
Can someone help me do it？

**Reply by nlohmann** [reply, marked as answer]
There is no API for this.

**Reply by sunshichao520** [reply]
Ok, thanks
